src/de/mod.rs

```diff
@@ -67,6 +67,22 @@ where
     Ok((t, deserializer.finalize()?))
 }
 
+/// Conveniently deserialize a message of type `T` from a byte slice with a Crc. The unused portion (if any)
+/// of the byte slice is not returned.
+///
+/// See the `de_flavors::crc` module for the complete set of functions.
+/// 
+#[cfg(feature = "use-crc")]
+pub use flavors::crc::from_bytes_u32 as from_bytes_crc32;
+
+/// Conveniently deserialize a message of type `T` from a byte slice with a Crc. The unused portion (if any)
+/// of the byte slice is returned for further usage
+///
+/// See the `de_flavors::crc` module for the complete set of functions.
+/// 
+#[cfg(feature = "use-crc")]
+pub use flavors::crc::take_from_bytes_u32 as take_from_bytes_crc32;
+
```

Comment:
Convenience functions. I"ve just provided the 32-bit width variants though. I figure that if the other variants are required then they can use the functions directly from the associated `flavors` package.

Reply:
That seems reasonable to me!